datadog_checks_dev/datadog_checks/dev/tooling/commands/release/testable.py

```diff
@@ -115,30 +155,22 @@ def testable(ctx, start_id, agent_version, milestone, dry_run):
         if result.code:
             abort(f'Unable to run {fetch_command}.')
 
-        if current_release_branch in result.stderr or diff_target_branch in result.stderr:
-            abort(
-                'Your repository is not sync with the remote repository. Please run git fetch in {} folder.'.format(
-                    root
-                )
-            )
+        if base_ref in result.stderr or target_ref in result.stderr:
+            abort(f'Your repository is not sync with the remote repository. Please run git fetch in {root!r} folder.')
 
-        # compare with the local tag first
-        reftag = f"{'refs/tags/'}{current_release_branch}"
-        result = run_command(diff_command.format(reftag, diff_target_branch), capture=True)
+        reftag = f"refs/tags/{base_ref}"
+        result = run_command(diff_command.format(reftag, target_ref), capture=True)
```

Comment:
This might be a good place to add an `echo_debug` to see the actual diff command being run.

Reply:
Addressed by outputting the output from the command in case of an error :-)